prusti-tests/tests/verify/fail/floats/nan_cmp_nan.rs

```diff
@@ -0,0 +1,40 @@
+// ignore-test: link to the issue
```

Comment:
Since `f32` works, could you please split this into two files: one for `f32` and one for `f64`?